### PR TITLE
docs aws: Add explicit note about cross-account S3 support

### PR DESCRIPTION
**Pull request by tazjin**
Relates to #353

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2462)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

File: docs/aws.md [hunk 1]
```diff
@@ -258,9 +258,14 @@ aws s3api put-bucket-versioning --bucket prefix-example-com-state-store  --versi
...
 
-If you configure a single S3 bucket to maintain kops state for clusters across
-multiple accounts, you may want to override the object ACLs which kops places
-on the state files.
+It is possible to use a single S3 bucket for storing kops state for clusters
+located in different accounts, by using [cross-account bucket policies](http://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example2.html#access-policies-walkthrough-cross-account-permissions-acctA-tasks).
+
+Kops will be able to use buckets configured with cross-account policies by default.
+
+In this case you may want to override the object ACLs which kops places on the
```
Comment (chrislovecnm) [inline docs/aws.md]: You have an example of this, or documentation on how to do this?

Comment (tazjin) [inline docs/aws.md]: Yes, it's in the next paragraph. Some of this was already documented :)

**Pull request merged**